Automattic/wp-calypso
# Social previews: transform JS files to TS files

## Proposed Changes

This PR transforms JS files at the root of the `social-previews` package to TS. Files for specific previews are handled separately (e.g., #75430 for the Facebook preview).

## Testing Instructions



* Review the code
* Make sure you have a self-hosted Jetpack site
* Spin up Calypso by using a live link below or by running this branch locally
* Open the dev console
* VIsit `/marketing/traffic/:site`
* Click _Show Previewd_ in the _Website Meta_ section, then select any preview
* Check there's no error in the console

## Pre-merge Checklist
